From: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Mon, 16 Mar 2026 10:31:00 +0800
Subject: man-test2

# DP: Document handling of parameters of the test builtin.

# DP: Document handling of parameters of the test builtin.
---
 builtins/test.def | 3 +++
 doc/bash.1        | 4 ++++
 doc/bashref.texi  | 4 ++++
 3 files changed, 11 insertions(+)

diff --git a/builtins/test.def b/builtins/test.def
index b6d5b81..2140b6a 100644
--- a/builtins/test.def
+++ b/builtins/test.def
@@ -100,6 +100,9 @@ Arithmetic binary operators return true if ARG1 is equal, not-equal,
 less-than, less-than-or-equal, greater-than, or greater-than-or-equal
 than ARG2.
 
+See the bash manual page bash(1) for the handling of parameters (i.e.
+missing parameters).
+
 Exit Status:
 Returns success if EXPR evaluates to true; fails if EXPR evaluates to
 false or an invalid argument is given.
diff --git a/doc/bash.1 b/doc/bash.1
index db1a957..ffb5c62 100644
--- a/doc/bash.1
+++ b/doc/bash.1
@@ -804,6 +804,10 @@ as primaries.
 .IP
 When used with \fB[[\fP, the \fB<\fP and \fB>\fP operators sort
 lexicographically using the current locale.
+.PP
+See the description of the \fItest\fP builtin command (section SHELL
+BUILTIN COMMANDS below) for the handling of parameters (i.e.
+missing parameters).
 .IP
 When the \fB==\fP and \fB!=\fP operators are used, the string to the
 right of the operator is considered a pattern and matched according
diff --git a/doc/bashref.texi b/doc/bashref.texi
index 138a0f9..f9c8fe3 100644
--- a/doc/bashref.texi
+++ b/doc/bashref.texi
@@ -8267,6 +8267,10 @@ The @code{test} command uses ASCII ordering.
 Unless otherwise specified, primaries that operate on files follow symbolic
 links and operate on the target of the link, rather than the link itself.
 
+See the description of the @code{test} builtin command (section
+@pxref{Bash Builtins} below) for the handling of parameters
+(i.e. missing parameters).
+
 @table @code
 @item -a @var{file}
 True if @var{file} exists.
